tests/e2e/replication/show_while_creating_invalid_state.py

```diff
@@ -1052,7 +1052,7 @@ def retrieve_data():
     # 5/
     expected_data = {
         ("sync_replica1", "127.0.0.1:10001", "sync", 0, 0, "invalid"),
-        ("sync_replica2", "127.0.0.1:10002", "sync", 5, 0, "ready"),
+        ("sync_replica2", "127.0.0.1:10002", "sync", 6, 0, "ready"),
```

Comment:
@gvolfing what does that number represents?

Reply:
`current_timestamp_of_replica`

Reply:
It has been nugded a couple of times in these past days.

Reply:
why exactly did that change here? 🤔

Reply:
To call `ApproximateVertexCount` we need a storage accessor, which will prop up the count.